client/views/admin/apps/AppsProvider.tsx

```diff
@@ -34,7 +35,7 @@ const AppsProvider: FC = ({ children }) => {
 	const [finishedLoading, setFinishedLoading] = useState<boolean>(() => false);
 
 	const ref = useRef(apps);
-	ref.current = apps;
+	// ref.current = apps;
```

Comment:
I was confused by the existence of this line and I commented it, so I could ask here if it really is doing the exact same thing of the line above or if I'm missing something 😛

Reply:
I`m 99.8% sure (maybe 99.9%) its some kind of "workaround/gambiarra" to use the same apps list in many places, I think not sure